### PR TITLE
[CLOUD-2235] add reference to kie-modules

### PR DESCRIPTION
**Pull request by jmtd**
https://issues.jboss.org/browse/CLOUD-2235

KIE modules are moving repositories; make sure we reference the new location.

- [✓] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [✓] Pull Request contains link to the JIRA issue
- [✓] Pull Request contains description of the issue
- [✓] Pull Request does not include fixes for issues other than the main ticket
- [✓] Attached commits represent units of work and are properly formatted
- [✓] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [✓] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

Signed-off-by: Jonathan Dowland <jdowland@redhat.com>

### CODE REVIEW

**Review by spolti — DISMISSED** [review]

**Review by errantepiphany — APPROVED** [review]
@jmtd Looks good to me.

File: image.yaml [hunk 1]
```diff
@@ -131,15 +131,18 @@ ports:
...
+              url: https://github.com/jboss-container-images/jboss-kie-modules.git
+              ref: master
       install:
-          - name: os-kieserver-launch
-          - name: os-kieserver-launch
-          - name: os-kieserver-probes
-          - name: os-kieserver-s2i
-          - name: os-kieserver-webapp
-          - name: os-kieserver-chmod
+          - name: os.kieserver.launch
```
Comment (rcernich) [inline image.yaml:140]: Can we use a more descriptive name, e.g. like Java package name for these?  The goal being to be able to easily identify from which repository the module is pulled, e.g. jboss.kie.kieserver.launch: kieserver.launch module from jboss-kie repsoitory.  If you have a better idea for naming, feel free to adjust as necessary.  With the example I provided, you might also want to add a name to the repo, e.g. jboss.kie.repo.  Having a name is a critical piece required to override the module location for development purposes, e.g. overrides.yaml:
```yaml
modules:
  repositories:
  - name: jboss.kie.repo
    path: /home/me/src/git/jboss-container-images/jboss-kie-modules
```

Comment (errantepiphany) [reply image.yaml:140]: @rcernich I like this idea (changing the names), but can we do this as follow-up PRs? There's already a lot in this jira...

Comment (errantepiphany) [reply image.yaml:140]: After speaking with @rcernich on IRC, the renaming of modules (and adding repo names) will be done in subsequent PRs. We need to move these PRs so other work can progress.

**Pull request merged**